### PR TITLE
sign-in-admin

### PR DESCRIPTION
**Pull request by bugs-2209**
## Related Tickets:
- [#41692](https://edu-redmine.sun-asterisk.vn/issues/41692)
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## WHAT this PR does?
- This PR is about login function for admin

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by bugs-2209** [comment]
ready

File: app/Http/Controllers/Admin/AuthController.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            return redirect()->route('admin.dashboard');
+        }
+
+        return redirect()->back();
+    }
+
+    public function logout()
+    {
+        Auth::logout();
+        return redirect()->route('admin.view_login');
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Admin/AuthController.php]: 1 line trống trc return

**Review by huyenttm-0901 — APPROVED** [review]

**Review by huyenttm-0901 — COMMENTED** [review]
comment

**Comment by bugs-2209** [comment]
ready

**Pull request merged**